options/options.go

```diff
@@ -45,6 +35,8 @@ var (
 
 const IncompatibleArgsErrorFormat = "illegal argument combination: cannot specify %s and --uri"
 
+const unknownOptionsWarningFormatString = "WARNING: ignoring unsupported URI parameter '%v'"
```

Comment:
```suggestion
const unknownOptionsWarningFormat = "WARNING: ignoring unsupported URI parameter '%v'"
```
[nit] no need to include the type in the name

Reply:
Sounds reasonable. Will do!